include/coreir/passes/analysis/vmodule.h

```diff
@@ -95,6 +95,7 @@ struct VModule {
   std::vector<std::string> stmts;
   VModules* vmods;
   string modComment = "";
+  string full_module = "";
```

Comment:
Maybe `module_verilog_string`?

Reply:
or something like `verilog_definition_string`

Reply:
how about `verilog_string`? We want to avoid "definition" because we generally use definition to mean the body. `module_verilog_string` seems to read a bit long.